std/socket.d

```diff
@@ -1749,16 +1749,19 @@ public:
     /// Any IPv6 host address.
     static @property ref const(ubyte)[16] ADDR_ANY() pure nothrow @nogc
     {
-        const(ubyte)[16]* addr;
         static if (is(typeof(IN6ADDR_ANY)))
         {
-            addr = &IN6ADDR_ANY.s6_addr;
```

Comment:
Wow, it was taking the address of an `enum` `struct` field. That shouldn't be allowed DIP1000 or not :)